src/components/Card/VideoCard/VideoCard.jsx

```diff
@@ -1,36 +1,63 @@
 import React from "react";
 import styles from "./videocard.module.css";
 import { BsThreeDotsVertical } from "react-icons/bs";
+import { MdDelete } from "react-icons/md";
+import { useLocation, Link } from "react-router-dom";
 import { getTrimVideoTitle } from "utils";
+import { useVideosOperations } from "context";
 
 export function VideoCard({
   video: {
+    _id: id,
     title,
     thumbnail,
+    videoID,
     views,
     creator: { profile: creatorProfile, name: creatorName },
   },
 }) {
+  const { manageDeleteHistory } = useVideosOperations();
+  const { pathname } = useLocation();
   const trimmedTitle = getTrimVideoTitle(title);
+  const btnDeleteHandler = () => {
+    manageDeleteHistory(id);
+  };
   return (
     <>
       <div className={`card ${styles.cardVideo}`}>
-        <img className='responsive-img' src={thumbnail} loading='lazy' alt='' />
-
-        <div className={`card-body flex-row ${styles.videoCardBody}`}>
+        <Link to={`/watch/${videoID}`}>
           <img
-            className='avatar avatar-round avatar-xs'
-            src={creatorProfile}
-            alt={creatorName}
+            className='responsive-img'
+            src={thumbnail}
+            loading='lazy'
+            alt=''
```

Comment:
suggestion - add dynamic alt text from the object, like the way you're adding img src

Reply:
Sure will make the changes.